packages/@uppy/compressor/src/index.js

```diff
@@ -46,8 +46,9 @@ export default class Compressor extends BasePlugin {
       async (file) => {
         try {
           const compressedBlob = await this.compress(file.data)
-          this.uppy.log(`[Image Compressor] Image ${file.id} size before/after compression: ${file.data.size} / ${compressedBlob.size}`)
-          totalCompressedSize += compressedBlob.size
+          const compressedSavingsSize = file.data.size - compressedBlob.size
+          this.uppy.log(`[Image Compressor] Image ${file.id} compressed by ${prettierBytes(compressedSavingsSize)}`)
+          totalCompressedSize += compressedSavingsSize
```

Comment:
We should probably rename that variable to something like `totalSavingsFromCompression`.